### ISSUE TITLE
A100 for training simtrack but not reproduce the paper results

### ISSUE THREAD

**Issue by gzgzgz666** [qcraftai/simtrack#30]
Dear author，I still have some questions.
1. We use A100 for training, 4 gpu cards, 4 epochs , the rest of the parameters have not changed, the training 200 epochs still does not achieve the effect. What training skills do you have? such as the learning rate modify and so on
2. We train the model that comes with simtrack. After training for 20 epochs, we continue to train based on the model, and the loss will continue to decline. why did you stop training?
I sincerely look forward to your reply.

**Comment by chenxuluo** [comment]
same as #29
